extensions/ql-vscode/src/variant-analysis/variant-analysis-manager.ts

```diff
@@ -848,7 +866,17 @@ export class VariantAnalysisManager
       this.app.logger,
       "Cancelling variant analysis. This may take a while.",
     );
-    await cancelVariantAnalysis(this.app.credentials, variantAnalysis);
+    try {
+      await cancelVariantAnalysis(this.app.credentials, variantAnalysis);
+    } catch (e) {
+      if (variantAnalysis.status === VariantAnalysisStatus.Canceling) {
+        await this.onVariantAnalysisUpdated({
```

Comment:
Shall we add a log message here saying we were unable to cancel the variant analysis?

Reply:
Isn't throwing the exception enough here?

Reply:
Maybe that's good enough. If the user sees an uncaught exception hopefully that clues them into the idea that something went wrong with canceling the analysis.